src/components/MultiSelect/MultiSelect.scss

```diff
@@ -1,10 +1,137 @@
 @use "sass:map";
 @import "vanilla-framework";
-@include vf-base;
-@include vf-p-lists;
 
 $dropdown-max-height: 20rem;
 
+// XXX: Temporarly we are duplicating some of the placeholders defined in Vanilla
```

Comment:
In longer term we should either upstream this component to Vanilla or refactor some bits of Vanilla to be able to extend these parts that are needed here, without the need to include the whole base or list styles, because duplication of base form styles was the cause of the issue (broken slider styles overriden by duplicated input styles).

Reply:
I'd suggest we link to a new issue in react-components where we explain this in detail (and move this comment there).

Reply:
It felt more fitting on Vanilla itself: https://github.com/canonical/vanilla-framework/issues/5030